tools/ci_connector_ops/ci_connector_ops/acceptance_test_config_checks.py

```diff
@@ -39,23 +40,36 @@ def find_connectors_with_bad_strictness_level() -> List[utils.Connector]:
                 connectors_with_bad_strictness_level.append(connector)
     return connectors_with_bad_strictness_level
 
-def find_changed_ga_connectors() -> List[utils.Connector]:
+def find_changed_ga_connectors() -> Set[str]:
     """Find GA connectors modified on the current branch.
 
     Returns:
-        List[utils.Connector]: The list of GA connectors that were modified on the current branch.
+        Set[str]: Set of connector names e.g {"source-github"}: The set of GA connectors that were modified on the current branch.
     """
     changed_connectors = utils.get_changed_connectors()
-    return [connector for connector in changed_connectors if connector.release_stage == "generally_available"]
+    return {connector.technical_name for connector in changed_connectors if connector.release_stage == "generally_available"}
```

Comment:
Could we make it continue to return and iterable of  `Connector`? This function might need to be reused elsewhere and will benefit from getting a full connector object

Reply:
Sure, I'll either make the regex one also return `Connector` objects, or will parse the `.technical_name` on the other side before comparing 👍🏻

Reply:
You're right, this was the better option! Done :) (made everything return Connector objects)